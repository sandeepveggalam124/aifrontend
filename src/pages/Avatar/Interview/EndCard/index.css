.finsih-link {
    color: #0E0E10;
}