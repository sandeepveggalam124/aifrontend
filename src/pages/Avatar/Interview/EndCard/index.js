import React, { useContext } from 'react'
import { Link } from 'react-router-dom'
import './index.css'
import { LoginContext } from '../../../../contexts/loginContext'

const EndCard = () => {
  const {logout}=useContext(LoginContext)
  // const handleClick=()=>{
  //   logout()
  // }
  return (
    <div className="card bg-black" style={{width:'25rem'}}>
    <div className="card-body d-flex flex-column justify-content-center align-items-center text-center">
      <p className="card-text">Thank you for completing your interview with us. We 
              appreciate your time and interest in our company. We 
              will be in touch with you regarding the next steps in our 
              hiring process. Have a great day!</p>
      <button className="btn btn-light fs-6 fw-medium">
        <Link to='/login' className='text-decoration-none text-dark'>
          Ok
        </Link>
      </button>
    </div>
  </div>
   
  )
}

export default EndCard