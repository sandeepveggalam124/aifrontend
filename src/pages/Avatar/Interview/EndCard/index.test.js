import { MemoryRouter as Router } from "react-router-dom";
import {render} from "@testing-library/react";
import EndCard from "./index";


it('renders correctly',()=>{
    const {container}=render(
        <Router>
            <EndCard/>
        </Router>
    )
    expect(container).toMatchSnapshot()
})