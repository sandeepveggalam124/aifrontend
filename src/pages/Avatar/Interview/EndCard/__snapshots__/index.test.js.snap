// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="card bg-black"
    style="width: 25rem;"
  >
    <div
      class="card-body d-flex flex-column justify-content-center align-items-center text-center"
    >
      <p
        class="card-text"
      >
        Thank you for completing your interview with us. We appreciate your time and interest in our company. We will be in touch with you regarding the next steps in our hiring process. Have a great day!
      </p>
      <button
        class="btn btn-light fs-6 fw-medium"
      >
        <a
          class="text-decoration-none text-dark"
          href="/"
        >
          Ok
        </a>
      </button>
    </div>
  </div>
</div>
`;
